meerk40t/gui/wxmmain.py

```diff
@@ -1143,6 +1310,27 @@ def register_options_and_choices(self, context):
         else:
             set_icon_appearance(1.0, 0)
 
+        self.format_painter = FormatPainter(self.context, "button/basicediting/Paint", "editpaint")
+        self.context.kernel.register(
+            "button/basicediting/Paint",
+            {
+                "label": _("Paint format"),
+                "icon": icon_paint_brush,
+                "tip": _("First select your template, then every subsequent selection will apply the templates properties to the selected elements"),
+                "help": "basicediting",
+                "action": self.format_painter.on_click,
+                "identifier": "editpaint",
+                "toggle": {
+                    "label": _("Stop"),
+                    "help": "devicegrbl",
```

Comment:
This can't be the correct help.

Reply:
I was following the design patterns of office solutions where you have that all the time, I could add a secondary ribbon panel, that's not visible that would contain the old config without the button, so that you could pick the leaner version in the ribbon editor